lib/puppet/type/ca.rb

```diff
@@ -5,4 +5,20 @@
 
   instance_eval(&Certs::CERT_COMMON_PARAMS)
 
+  newparam(:other_certs) do
```

Comment:
```:array_matching => :all``` might be a good one to add https://docs.puppet.com/puppet/5.0/custom_types.html#customizing-behaviour